### PR TITLE
PMM-1797 fix CloudWatch DataSource editing

### PR DESCRIPTION
**Pull request by delgod**

### CODE REVIEW

File: import-dashboards.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def add_datasources():
             sys.exit(-1)
 
     if 'CloudWatch' not in ds:
-        data = json.dumps({'name': 'CloudWatch', 'type': 'cloudwatch', 'jsonData': '{"defaultRegion":"us-east-1"}', 'access': 'proxy', 'isDefault': False})
+        data = '{"name":"CloudWatch","type":"cloudwatch","url":"","access":"proxy","jsonData":{"timeField":"@timestamp","authType":"keys","defaultRegion":"us-east-1"},"secureJsonFields":{}}'
```
Comment (AlekSi) [inline import-dashboards.py]: Any particular reason why you don't use `dict` and `json.dumps()`?

Comment (delgod) [reply import-dashboards.py]: it is fast proof-of-concept, feel free to rework code as you wish for your ticket ;)

**Comment by rnovikovP** [comment]
looks like we need this only until 1.6 release because of  https://jira.percona.com/browse/PMM-1063, so no need to have a good code. just some fix - will be ok.

**Comment by AlekSi** [comment]
See #38.

**Pull request closed without merging**